test/JDBC/upgrade/latest/schedule

```diff
@@ -534,3 +534,4 @@ test_conv_string_to_datetime2
 test_conv_string_to_datetimeoffset
 test_conv_string_to_smalldatetime
 test_conv_string_to_time
+BABEL-3401
```

Comment:
Shouldn't we add this in 14_12/schedule as well?

Reply:
Added